pkg/common/utils/resource/pod.go

```diff
@@ -678,6 +688,50 @@ func getMultiConfigVolumeAndVolumeMount(cmInfo *v1.ConfigMapInfo, componentType
 	return volumes, volumeMounts
 }
 
+func getMultiSecretVolumeAndVolumeMount(bSpec *v1.BaseSpec, componentType v1.ComponentType) ([]corev1.Volume, []corev1.VolumeMount) {
+	var volumes []corev1.Volume
+	var volumeMounts []corev1.VolumeMount
+
+	if bSpec.Secrets == nil {
+		return volumes, volumeMounts
+	}
+
+	defaultMountPath := ""
+	switch componentType {
+	case v1.Component_FE, v1.Component_BE, v1.Component_CN, v1.Component_Broker:
+		defaultMountPath = config_env_path
```

Comment:
can't use env_config_path,should use secret_config_path

Reply:
Done